recipe/meta.yaml

```diff
@@ -86,7 +88,8 @@ requirements:
     - utf8proc
     # It's not clear to me, why conda-build detects them, but doesn't create
     # an automatic dependency.
-    - zlib  # [win]
+    # - zlib  # [win]
+    - zlib
```

Comment:
Which do we need to explicitly list `zlib` as a run entry on *nix? That should have been taken care of by the run exports.